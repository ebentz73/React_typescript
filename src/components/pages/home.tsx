// @flow
import React from 'react';
import {styled} from 'fusion-plugin-styletron-react';

const Container = styled('div', {
  height: '100%',
  textAlign: 'center',
  backgroundColor: '#FFFFFF',
});

export const Home = () => <Container>Frosted Affair</Container>;
