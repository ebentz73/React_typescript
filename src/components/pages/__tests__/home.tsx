import React from 'react';
import {shallow} from 'enzyme';
import {Home} from '../home';

test('Home', () => {
  expect(shallow(<Home />)).toMatchSnapshot();
});
