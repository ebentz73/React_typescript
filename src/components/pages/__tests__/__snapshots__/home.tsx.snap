// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home 1`] = `
<Styled(div)>
  Frosted Affair
</Styled(div)>
`;
